src/Core/src/HotReload/HotReloadHelper.cs

```diff
@@ -51,6 +52,8 @@ public static bool IsReplacedView(IHotReloadableView view, IView newView)
 				return false;
 			return newView.GetType() == newViewType;
 		}
+
+		[UnconditionalSuppressMessage("Trimming", "IL2062", Justification = "Hot Reload is not supported in Release (trimmed) scenarios.")]
 		public static IView GetReplacedView(IHotReloadableView view)
```

Comment:
This is the only "Hot Reload"-related method that is not actually linked away in a `Release` build, There might be a way to fix this so it *is* linked away. For now, it seems like 1 attribute to ignore this would be fine?

Reply:
There is a feature switch available which you could use for it as well, it's called `System.Reflection.Metadata.MetadataUpdater.IsSupported`